test/client/data/style-sandbox/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="/hammerhead.js" class="script-hammerhead-shadow-ui"></script>
+</head>
+<body>
+<div id="item" style="background: url('https://testcafe.io/img/qa-guy.svg');width: 500px;height: 500px;"></div>
```

Comment:
Don't use links to outer sites. Create a simple image in Paint and use it in tests.